### ISSUE TITLE
FormKitSchema does not work and gives error when  autoImport is set to True in nuxt.config.ts

### ISSUE THREAD

**Issue by Subfortytwo** [formkit/formkit#995]
### Reproduction

https://formkit.link/b30180460637fca8bd9f535940e359f9

### Describe the bug

When  `$formkit` is used in the schema and autoImport is set to true, browser complains and does not print the required element. 
Here is an example:

```
<script setup>
import { FormKitSchema } from '@formkit/vue'

const data = reactive({ value: 0 })
const schema = [
  {
    $el: 'h2',
    if: '$value >= 5 && $value <= 10',
    children: '$value + " is between 5-10!"',
  }, 
  {
    $formkit: "checkbox",
    name: "hasEmail",
    label: "Has Email?",
  }
]
</script>

<template>
  <FormKit
    type="number"
    v-model="data.value"
    label="Enter a number between 5-10"
  />
  <FormKitSchema
    :schema="schema"
    :data="data"
  />
</template>
```

The browser will complain:

```
[Vue warn]: Failed to resolve component: FormKit
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
```

### Environment

```
  "devDependencies": {
    "nuxt": "^3.6.5",
    "nuxt-windicss": "^2.6.1"
  },
  "dependencies": {
    "@formkit/icons": "^1.2.2",
    "@formkit/nuxt": "^1.2.2",
    "@formkit/themes": "^1.2.2",
    "@vee-validate/nuxt": "^4.11.7",
    "mitt": "^3.0.1",
    "nuxi": "^3.6.5",
    "vite": "^4.4.11",
    "vue": "^3.3.4",
    "yarn": "^1.22.19"
  }

```

**Comment by jetlej** [comment]
I'm getting this too. Is there an interim solution until this is merged?

**Comment by Subfortytwo** [comment]
Set autoimport to false :P.

**Comment by justin-schroeder** [comment]
This should be fixed in 1.3.0
